lib/Manager.php

```diff
@@ -173,6 +173,7 @@ public function createRoomObject(array $row): Room {
 			(string) $row['name'],
 			(string) $row['description'],
 			(string) $row['password'],
+			(string) $row['avatar'],
```

Comment:
you cast to string here, but the setter allows null, so just allow it here too:
```suggestion
			$row['avatar'],
```